tools/docker/coverage.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+###############################################################
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+# SPDX-FileCopyrightText: 2019-2020 the prplMesh contributors (see AUTHORS.md)
+# This code is subject to the terms of the BSD+Patent license.
+# See LICENSE file for more details.
+###############################################################
+
+scriptdir="$(cd "${0%/*}" || exit 1; pwd)"
+rootdir="${scriptdir%/*/*}"
+
+# shellcheck source=functions.sh
+. "${rootdir}/tools/functions.sh"
+
+main() {
+    image=${PRPLMESH_BUILDER_IMAGE-"${DOCKER_REGISTRY}"prplmesh-builder}
+    # Default docker arguments
+    docker_args=(
+        --workdir "${rootdir}"
+        --user "${SUDO_UID:-$(id -u)}:${SUDO_GID:-$(id -g)}"
+        -e "USER=${SUDO_USER:-${USER}}"
+        -v "${rootdir}:${rootdir}"
+        --name prplMesh-coverage
+        --entrypoint "./coverage.sh"
+    )
+
+    run docker container run "${docker_args[@]}" --rm "${image}" build "$@"
```

Comment:
Why is this "build" argument kept? It looks like it's not used anywhere?

Since this script doesn't really add anything different than the clang-format one for example, we could refactor it in a single script (builder.sh for example) that would just run the command you pass it inside the builder image, with the docker options already set.

IMO it's ok if you want keep this for now though.

Reply:
Removed "build"